### DISCUSSION TITLE
Supabase local development bucket not working with nextimage

### DISCUSSION THREAD

**Discussion by rndy28** [supabase/supabase#14961] [Questions] [answered]
Hi there, iam trying to show image with nextimage from public bucket, but nextjs throwing this error:

`Error: Invalid src prop (postgresql://postgres:postgres@localhost:54322/postgres/storage/v1/object/public/assets/covers/no_longer_human-osamu_dazai) on `next/image`, hostname "" is not configured under images in your `next.config.js`
See more info: https://nextjs.org/docs/messages/next-image-unconfigured-host`

I already configure domain like this, the warning is gone but the image is not showing
```javascript
images: {
      domains: ["localhost", ""],
    },
```

here's the func to get image:

```typescript
export function getSupabaseAssets(path: string): string {
  const {
    data: { publicUrl },
  } = supabase.storage.from("assets").getPublicUrl(path);

  return publicUrl;
}
```

this is how i called it

```javascript
<Image
  src={getSupabaseAssets(`covers/${props.cover}`)}
  alt={`${props.authors} - ${props.title}`}
  width={400}
  height={600}
  className="rounded-tr-md rounded-tl-md"
/>
```

and here's the bucket

![2023-06-11-170643_876x673_scrot](https://github.com/supabase/supabase/assets/61275105/f5004902-23aa-4020-bbf9-c363a2527056)

**Comment by GaryAustin1** [comment, marked as answer]
I don't do local dev, but your url looks wrong.   It appears to be the database direct port URL versus a local http address...
`postgresql://postgres:postgres@localhost:54322/postgres`  this looks wrong.
https://github.com/supabase/storage-api

**Reply by rndy28** [reply]
> I don't do local dev, but your url looks wrong. It appears to be the database direct port URL versus a local http address... `postgresql://postgres:postgres@localhost:54322/postgres` this looks wrong. https://github.com/supabase/storage-api

Hi, thanks for answering. Yeah, I notice I'm putting the wrong value for SUPABASE_URL env, thanks.
